polytiles/polytiles.py

```diff
@@ -529,88 +546,88 @@ def poly_parse(fp):
     else:
-        cur.execute("""SELECT ST_Transform(ST_Union(ST_Buffer(p.way, 5000)), 4326) FROM planet_osm_point p WHERE p.place in ('town', 'city') AND NOT EXISTS(SELECT 1 FROM planet_osm_polygon pp WHERE pp.name=p.name AND ST_Contains(pp.way, p.way));""")
+        cur.execute("SELECT ST_Transform(ST_Union(ST_Buffer(p.way, 5000)), 4326) FROM planet_osm_point p WHERE p.place in ('town', 'city') AND NOT EXISTS(SELECT 1 FROM planet_osm_polygon pp WHERE pp.name=p.name AND ST_Contains(pp.way, p.way));")
     result = cur.fetchone()
     if result:
-        poly = poly.union(loads(result[0].decode('hex')))
+        poly = poly.union(loads(result[0].decode("hex")))
     return poly
 
 
 def main():
     try:
-        mapfile = os.environ['MAPNIK_MAP_FILE']
+        mapfile = os.environ["MAPNIK_MAP_FILE"]
     except KeyError:
-        mapfile = os.getcwd() + '/osm.xml'
+        mapfile = os.getcwd() + "/osm.xml"
 
     default_user = getpass.getuser()
 
-    parser = argparse.ArgumentParser(description='Generate mapnik tiles for an OSM polygon')
-    apg_input = parser.add_argument_group('Input')
-    apg_input.add_argument("-b", "--bbox", nargs=4, type=float, metavar=('X1', 'Y1', 'X2', 'Y2'), help="generate tiles inside a bounding box")
-    apg_input.add_argument('-p', '--poly', type=argparse.FileType('r'), help='use a poly file for area')
+    parser = argparse.ArgumentParser(description="Generate mapnik tiles for an OSM polygon")
+    apg_input = parser.add_argument_group("Input")
+    apg_input.add_argument("-b", "--bbox", nargs=4, type=float, metavar=("X1", "Y1", "X2", "Y2"), help="generate tiles inside a bounding box")
+    apg_input.add_argument("-p", "--poly", type=argparse.FileType("r"), help="use a poly file for area")
     if HAS_PSYCOPG:
-        apg_input.add_argument("-a", "--area", type=int, metavar='OSM_ID', help="generate tiles inside an OSM polygon: positive for polygons, negative for relations, 0 for whole database")
-        apg_input.add_argument("-c", "--cities", type=int, metavar='OSM_ID', help='generate tiles for all towns inside a polygon')
-    apg_input.add_argument('-l', '--list', type=argparse.FileType('r'), metavar='TILES.LST', help='process tile list')
-    apg_output = parser.add_argument_group('Output')
-    apg_output.add_argument('-t', '--tiledir', metavar='DIR', help='output tiles to directory (default: {0}/tiles)'.format(os.getcwd()))
-    apg_output.add_argument('--tms', action='store_true', help='write files in TMS order', default=False)
+        apg_input.add_argument("-a", "--area", type=int, metavar="OSM_ID", help="generate tiles inside an OSM polygon: positive for polygons, negative for relations, 0 for whole database")
```

Comment:
This change of quotes has really messed this pull request up. I hope you did have a reason.